creator/settings/features.py

```diff
@@ -158,6 +158,15 @@
 )
 
 
+# GWO INGEST RUNS ##############################################################
+# The Study Creator can automate various ingest processes such as ingesting
+# harmonized genomic files.
+
+FEAT_INGEST_GENOMIC_WORKFLOW_OUTPUTS = os.environ.get(
+    'FEAT_INGEST_GENOMIC_WORKFLOW_OUTPUTS', 'True'
```

Comment:
I think black should format this with double quotes?

Reply:
that's true. what black options should i use in general? i notice that when you just run `black` it sometimes makes lines too long

Reply:
Black should always format consistently except for those weird cases where it won't adjust the line length for you. In those situations I usually have to break up the line manually and then let black format it nicely.